app/templates/deck/build.hbs

```diff
@@ -33,32 +32,21 @@
       {{/if}}
     </div>
     <div class="col-sm-5">
-      <form class="form-horizontal">
+      <form class="form-horizontal form-horizontal-condensed">
         <div class="form-group">
-          <label class="col-sm-2"></label>
-          <div class="col-sm-10">
-            <div class="btn-group">
-              <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                {{fa-icon "cog"}}
-                {{fa-icon "caret-down"}}
-              </button>
-              <ul class="dropdown-menu">
-                <li><a {{action "openModal" "deck/export-modal" model}}>Export Deck</a></li>
-                <li><a {{action "saveDeck" model}}>Save Deck</a></li>
-              </ul>
-            </div>
-            <span>Deck Classification: <span>{{controllers.deck.model.classification}}</span></span>
+          <label class="col-sm-6 control-label" style="text-align: left;">Classification</label>
+          <div class="col-sm-6" style="padding-top: 7px;">
```

Comment:
electing not to fix this with css since the selector is very specific and this is a one-off fix. if we incorporate more forms that don't have input fields then i will fix it more wholly then